q2_metadata/templates/tabulate/index.html

```diff
@@ -3,22 +3,59 @@
 {% block content %}
   <div class="row">
     <div class="col-lg-12">
-      <p>Export options:</p>
-      <table id='table' class='table table-hover'></table>
+      <p>
+        <a href="metadata.tsv" target="_blank" rel="noopener noreferrer" class="btn btn-default">
+          Download normalized (and if applicable, merged) metadata
+        </a>
+        <br>
+        This file won't necessarily reflect dynamic sorting options based on
+        the interactive table below.
+      </p>
+      <table id="table" class="table table-hover"></table>
+      <div id="loading" class="spinner">
+        <div class="rect1"></div>
+        <div class="rect2"></div>
+        <div class="rect3"></div>
+        <div class="rect4"></div>
+        <div class="rect5"></div>
+      </div>
     </div>
   </div>
   <link rel="stylesheet" type="text/css" href="./css/datatables.min.css"/>
   <script type="text/javascript" src="./js/datatables.min.js"></script>
 
   <script type="text/javascript">
     $(document).ready(function(){
-      var data = JSON.parse('{{ table }}');
+      var loading = $('#loading');
+
+      function renderErrorMsg(errors) {
+        var helpMsg = $('<div></div>');
+        helpMsg.append('<p class="alert alert-danger">\
+                        There was an error loading and rendering the \
+                        interactive table. You can download the normalized \
```

Comment:
"You can download the metadata TSV file above and load in an external application such as Microsoft Excel or Google Sheets. The metadata file can also be used anywhere metadata is accepted in QIIME 2."

(Just wanted to highlight that the file can still be used with QIIME 2, in addition to other programs)